pkg/fetch/ubuntu/oval/oval.go

```diff
@@ -123,7 +123,7 @@ func Fetch(opts ...Option) error {
 		log.Printf("[INFO] Fetch Ubuntu CVE %s/%s Definitions", release, service)
 		bar := pb.StartNew(len(r.Definitions.Definition))
 		for _, def := range r.Definitions.Definition {
-			if err := util.Write(filepath.Join(options.dir, release, service, "definitions", fmt.Sprintf("%s.json", def.ID)), def); err != nil {
+			if err := util.Write(filepath.Join(options.dir, release, "cve", service, "definitions", fmt.Sprintf("%s.json", def.ID)), def); err != nil {
 				return errors.Wrapf(err, "write %s", filepath.Join(options.dir, release, service, "definitions", fmt.Sprintf("%s.json", def.ID)))
```

Comment:
"cve" needed?

Reply:
I prefer the directories to be cve/main, cve/fips-updates, usn, pkg/main, pkg/fips-updates rather than main, fips-updates, usn, pkg/main, pkg/fips-updates.

Reply:
If you don't think so, then this isn't necessary.

Reply:
fixed https://github.com/MaineK00n/vuls-data-update/compare/f16656c21718c9ccf075df0c3e35348f1d670845..1aa0866404c5a4819fa892a9963bfcd608c41815